create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+	hosting category added) and provision it in the cloud
+	```
+	(Note: `cognito0cc0c000` will always include a UUID generated by Amplify so substitute it for yours in the following steps)
+	
+4. The new resources have only been configured in our local environment so let's publish them to our AWS Account: 
+	```
+	amplify push
+	| Category | Resource name   | Operation | Provider plugin   |
+	| -------- | --------------- | --------- | ----------------- |
+	| Auth     | cognito0xx0x000 | Create    | awscloudformation |
+	? Are you sure you want to continue? Yes
+	```
+5. You might get an error like this:
+	```
+	CREATE_FAILED UserPoolClientLambda                                  AWS::Lambda::Function
+	      
+	Wed Apr 07 2021 16:50:55 GMT-0400 (Eastern Daylight Time) The runtime parameter of 
+	nodejs6.10 is no longer supported for creating or updating AWS Lambda functions. 
+	We recommend you use the new runtime (nodejs12.x) while creating or updating functions.
+	```
+	In which case we can manually edit the CloudFormation template generated by Amplify to replace all occurrences of `nodejs6.10` with `nodejs12` either in our IDE or with the command line as shown below:
+	```
+	cd ~/.../create-custom-map-style/amplify/backend/auth/cognito0cc0c000
+	sed -i -e 's/nodejs6.10/nodejs12.x/g' cognito0cc0c000-cloudformation-template.yml
+	cd ~/blog/create-custom-map-style
+	amplify push
+	```
+6. After the Auth resource are created you should notice an autogenerated `aws-exports.js` in your project's `src` directory
+	
+### React-Amplify Integration
+1. Now that we've set up all the Authorization infrastructure needed for our application, we can test it out with our frontend React application. To start we need to add the `aws-amplify` package to our dependencies
+	```
+	npm i aws-amplify
+	```
+
+2. Open `src/index.js` in your IDE 
+
+3. Under your existing list of import statements create a new line and add the following: 
+	```
+	...
+	import  Amplify  from  "aws-amplify";
+	import  awsExports  from  "./aws-exports";
+	```
+
+4. Next we'll need to configure our Amplify so above the `ReactDOM.render` line let's add the following: 
+	```
+	...
+	Amplify.configure(awsExports);
+	...
+	```
+	The `configure()` method is just setting the AWS resources that we want to use for our backend. It might look intimidating, but just remember this isn’t doing anything special here beside configuration.
+
+5. At this point your `src/index.js` file should something like this:
+	```
+	import  React  from  'react';
+	import  ReactDOM  from  'react-dom';
+	import  './index.css';
+	import  App  from  './App';
+	import  reportWebVitals  from  './reportWebVitals';
+	import  Amplify  from  'aws-amplify';
+	import  awsExports  from  './aws-exports';
+
+	Amplify.configure(awsExports);
+	
+	ReactDOM.render(
+		<React.StrictMode>
+		  <App  />
+		</React.StrictMode>,
+		document.getElementById('root')
+	);
```

Comment:
What's up with the double spacing?

Reply:
must have gotten messed up in the markdown editor we were using, will change it 👍